integration/auth/user_mgt_test.go

```diff
@@ -51,13 +51,31 @@ func TestUserManagement(t *testing.T) {
 		{"Add custom claims", testAddCustomClaims},
 		{"Delete test users", testDeleteUsers},
 	}
+	// The tests are meant to be run in sequence, a failure in creating the users
+	// should be fatal so non of the other tests run. However calling Fatal from a
+	// subtest does not prevent the other subtests from running, hence we check the
+	// success of each subtest before proceeding.
 	for _, run := range orderedRuns {
 		if ok := t.Run(run.name, run.testFunc); !ok {
 			t.Fatalf("Failed run %v", run.name)
 		}
 	}
 }
 
+func forceCreateUser(uid string, params *auth.UserToCreate) (*auth.UserRecord, error) {
```

Comment:
This looks hacky. Can we remove this from this PR? We should implement a better solution in a separate PR.